uber/templates/macros.html

```diff
@@ -82,6 +82,36 @@
 {%- endmacro %}
 
 
+{% macro form_group_scaffolding(
+    label=False,
+    help=False,
+    is_required=False,
+    size=6,
+    form_class='',
+    static_wrap=False) -%}
```

Comment:
Yup, this is mostly just a convenience thing. In the form_checkgroup macro we had an `is_wrapped` variable that we then used like:
```
{% if is_wrapped %}<div class="form-control-static">{% endif %}
lots of stuff
{% if is_wrapped %}</div>{% endif %}
```
Now we still have the `is_wrapped` variable in the calling function, it just passes that as the `static_wrap` argument. I thought it was a bit cleaner to do it that way, but if you think it's unintuitive I can change it back.